packages/components/src/components/amount-input/__tests__/amount-input.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
`nit-non-blocking` shouldn't the test file naming conventions be `.spec` and not test